openfisca_us/parameters/gov/states/md/tax/income/married.yaml

```diff
@@ -0,0 +1,46 @@
+description: MD State income tax rate for single filers
+metadata:
+  type: single_amount
+  threshold_unit: currency-USD
+  amount_unit: currency-USD
+  label:
+  reference:
+    - title: Tax Foundation State Individual Income Tax Rates and Brackets
+      href: https://taxfoundation.org/publications/state-individual-income-tax-rates-and-brackets/#Structures
+      # So far still need to add the addition amounts below, will need to create variables
+    - title: MD Tax_Computation
+      href: https://www.marylandtaxes.gov/forms/current_forms/Md_Tax_Computation.pdf
+
+brackets:
+  - threshold:
+      2015-01-01: 0
+    amount:
+      2015-01-01: 0.02
```

Comment:
The source shows 0.0475 as the bottom rate

Reply:
True, this source was only for incomes above 100,000. Fixed